lib/mongoid/config/options.rb

```diff
@@ -36,8 +36,16 @@ def option(name, options = {})
             end
           end
 
-          define_method("#{name}=") do |value|
-            settings[name] = value
+          if name.to_sym == :broken_view_options
+            Mongo.send("#{name}=", options[:default])
+            define_method("#{name}=") do |value|
+              Mongo.send("#{name}=", value)
+              settings[name] = value
```

Comment:
If possible I wouldn't have a setting in Mongoid at all, this way the two can't get out of sync.

Reply:
Well, I want the default to be true in Mongoid and false in mongo-ruby-driver.... how else would you do that?